calculator.js

```diff
@@ -1,10 +1,91 @@
+/*jshint esversion: 6 */
 /**
  * Declare a function named `calculatorModule`
  * this function will have two private variables declared inside of it.
  * @variable PRIVATE { Number } `memory`
  * @variable PRIVATE { Number } `total`
  * @return {object} `calculator` object that can be used
  */
+function calculatorModule(){
+  let memory = 0;
+  let total = 0;
+
+  function load(x){
```

Comment:
Refactor your validation into a private function that can be used multiple times.